.github/workflows/build.yml

```diff
@@ -0,0 +1,149 @@
+name: Build
+env:
+  GEOS_VERSION: "3.6.5"
+
+on:
+  push:
+    paths:
+      - ".github/workflows/**"
+      - "packages/basemap/**"
+      - "packages/basemap_data/**"
+      - "packages/basemap_data_hires/**"
+  pull_request:
+    paths:
+      - ".github/workflows/**"
+      - "packages/basemap/**"
+      - "packages/basemap_data/**"
+      - "packages/basemap_data_hires/**"
+  workflow_dispatch:
+
+jobs:
+  build_data:
+    name: Build data packages
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package: [basemap_data, basemap_data_hires]
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
```

Comment:
Since the PR effectively moves the support to start with Python 3.9, we can also use Python 3.9 here (the minimum supported Python version).